src/api/release_tokens.js

```diff
@@ -0,0 +1,13 @@
+const { post } = require('request-promise')
+
+module.exports = (efx, tokein) => {
+  const url = efx.config.api + '/releaseTokens'
+
+  const data = {
+    address: '0x01...',
+    tokenAddress: '0x02...',
+    unlockUntil: 123456
```

Comment:
@hems I don't think we want unlockUntil to be specifiable - we will just give the user 10 minutes or so from whenever they submit the request. (Ignore the readme which is outdated in this case)

Reply:
@plutoegg okay then, I'm removing the unlock until value from the payload